tests/data/test_biased_random_walker.py

```diff
@@ -159,27 +159,13 @@ def test_parameter_checking(self):
         with pytest.raises(ValueError):
             # weighted is boolean which is by default False. It is True if walk has to be weighted.
             biasedrw.run(
-                nodes=nodes,
-                n=n,
-                p=p,
-                q=q,
-                length=length,
-                seed=seed,
-                weighted="unknown",
-                edge_weight_label="weight",
+                nodes=nodes, n=n, p=p, q=q, length=length, seed=seed, weighted="unknown"
             )
 
         with pytest.raises(ValueError):
             # edge weight labels are by default called weight as is in networkx but they can be any string value if user specified
```

Comment:
Probably. SEP if you read HHGTTG.

Reply:
It's definitely the problem of a PR that deletes/moves some functionality to remove outdated tests, otherwise there's just an endless accrual of weird and wacky tests that aren't doing anything useful. Fortunately this particular one isn't too entangled and we can just delete the lines.

Reply:
Ok, if this is a hang-over from me shifting arguments out of methods into the constructor, then I agree it is my problem. Will review.

Reply:
Still not convinced I made any changes relevant to this code, but I removed the duplicate test and reworded the comment.

Reply:
👍  (You deleted the `edge_weight_label` argument, which this second test was attempting to validate.)